app/views/maintenances/index.html.haml

```diff
@@ -0,0 +1,4 @@
+.autopagerize_page_element
+  - @maintenances.each do |maintenance|
+    %p= link_to maintenance.name, maintenance
+= paginate @maintenances, theme: 'twitter-bootstrap-4'
```

Comment:
Is there a possibility to set the theme default in kaminari_config.rb?

Reply:
I'm not sure just config file comes with
Kaminari.configure do |config|
  #config.default_per_page = 15
  # config.max_per_page = nil
  # config.window = 4
  # config.outer_window = 0
  # config.left = 0
  # config.right = 0
  # config.page_method_name = :page
  # config.param_name = :page
  # config.params_on_first_page = false
end

I'll have a look on documentation.